ethereum/executionclient/src/main/java/tech/pegasys/teku/ethereum/executionclient/web3j/Web3JClient.java

```diff
@@ -103,15 +109,18 @@ protected void handleError(final Throwable error, final boolean couldBeAuthError
             });
     if (maybeUpdatedTime == timeNow) {
       logExecutionClientError(error, couldBeAuthError);
+      executionClientEventsPublisher.onAvailabilityUpdated(false);
```

Comment:
This call means we will publish events for subsequent failures but this is probably ok as we don't expect our EL to be failing that much ;)